pkg/clusters/utils.go

```diff
@@ -75,3 +77,36 @@ func GetIngressLoadbalancerStatus(ctx context.Context, c Cluster, namespace stri
 		return nil, fmt.Errorf("%T is not a supported ingress type", ingress)
 	}
 }
+
+// CreateNamespace create customized namespace
+func CreateNamespace(ctx context.Context, cluster Cluster, namespace string) error {
+	nsName := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: namespace,
+		},
+	}
+	attempts := 5
+	for attempts > 0 {
+		_, err := cluster.Client().CoreV1().Namespaces().Create(context.Background(), nsName, metav1.CreateOptions{})
+		if err != nil && !errors.IsAlreadyExists(err) {
+			time.Sleep(1 * time.Second)
+			attempts--
+		}
```

Comment:
for this retry logic, it looks like you may want a `continue` statement here, am I wrong?

```suggestion
			attempts--
			continue
		}
```

Reply:
965d588392a9459c1338eac71d5af1123beea927